### ISSUE TITLE
Default shortcut not working in firefox

### ISSUE THREAD

**Issue by contrun** [zotero/zotero-connectors#433]
The default shortcut for firefox is not working as `Ctrl+Shift+S` is already bind to screenshot on firefox.
https://github.com/zotero/zotero-connectors/blob/9ad4dd1c1e2a12b7a738233c308c564d91b9649c/src/browserExt/manifest.json#L70

Can we change it to another shortcut so that the default shortcut works for firefox?

**Comment by hermanhel** [comment]
On firefox, go to manage extension shortcuts at `about:addons`
![image](https://github.com/zotero/zotero-connectors/assets/74124690/87f2897a-c9cf-4ce0-98ad-0f1cc0a9d4eb)

![image](https://github.com/zotero/zotero-connectors/assets/74124690/0e3824e4-0661-43e7-9c1b-8a1517f5e8e0)

And you can change it to whatever you want.

**Comment by contrun** [comment]
That's true and thanks for the information. I think change the default shortcut for firefox would be more preferable, as that simply does not work. Can we add some configuration for that in the code base. My use case originally was to do some headless scripting to zotero (save websites to zotero without gui interaction). I didn't find an easy way to change shortcut without GUI.

**Comment by dstillman** [comment]
(But on Chrome/Edge we override the built-in shortcut, at least for me on macOS.)
